packages/terra-clinical-item-view/tests/wdio/item-view-spec.js

```diff
@@ -0,0 +1,46 @@
+/* global before, browser, Terra */
+
+const viewports = Terra.viewports('tiny', 'small', 'medium', 'large', 'huge', 'enormous');
+const selector = '#root';
+
+describe('Item View', () => {
+  describe('Displays a clinical item view with default props', () => {
```

Comment:
This example can be removed. Jest test handles this adding these screenshots don't seem to add much value.